### PR TITLE
[Ready for review] Add form functionality for volunteering opportunity update.

### PR DESCRIPTION
**Pull request by nikhitashanker**
Add form functionality for volunteering opportunity update.

### CODE REVIEW

File: project/src/main/webapp/update-volunteering-opportunity.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            <input
+              id="required-skill"
+              type="text"
+              name="required-skill"
+              placeholder="Enter a skill"
+              class="form-control"
+            />
+          </div>
+        </div>
+        <button type="button" id="add_row" class="btn btn-default pull-left">
```
Comment (dash102) [inline project/src/main/webapp/update-volunteering-opportunity.html:62]: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Use hyphen to separate rows for ids and classes

Comment (nikhitashanker) [reply project/src/main/webapp/update-volunteering-opportunity.html]: Done.

**Comment by nikhitashanker** [comment]
[Ready for review build will be fixed by Events PRs]

**Comment by yicyao** [comment]
LGTM

**Pull request merged**